crates/html-validation/src/lib.rs

```diff
@@ -30,4 +130,22 @@ lazy_static! {
 /// ```
 pub fn is_self_closing(tag: &str) -> bool {
     SELF_CLOSING_TAGS.contains(tag)
+        || SVG_NAMESPACED_TAGS
+            .get(tag)
+            .map(|v|*v)
+            .unwrap_or(false)
+}
+
+/// Return the namespace of this tag
+/// returns the svg namespace as of now,
+/// any other tags will have a `None` value
+pub fn get_namespace(tag: &str) -> Option<String> {
```

Comment:
`Option<String>` feels a little off to me here since every tag has a namespace.

For now can we instead have a `is_svg_namespace (tag: &str) -> bool` and make use of that in our `create_element_node` function? This way we don't need to say that `None` really means HTML

Reply:
Yep, that make sense.